pkg/csi/cinder/nodeserver.go

```diff
@@ -526,6 +526,17 @@ func (ns *nodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
 	}
 	volumePath := req.GetVolumePath()
+	if len(volumePath) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Volume path not provided")
+	}
+
+	_, err := ns.Cloud.GetVolume(volumeID)
```

Comment:
@jichenjc From the issue details this is not required. 
https://github.com/kubernetes-csi/csi-test/blob/master/pkg/sanity/node.go#L682 , this checks for volumeID , which we are already doing in L525. So ideally shouldnt fail. Could you check again why that test case is failing.

Reply:
my bad, actually the test case is https://github.com/kubernetes-csi/csi-test/blob/master/pkg/sanity/node.go#L716
I updated issue just now

Reply:
@jichenjc got it. I think this is not caught by current test as we are using old version of csi-test . Could you update the csi-test to new version as well to make sure this part of code is tested in CI as well. Thanks.